vprof/base_profile.py

```diff
@@ -1,5 +1,7 @@
 """Base class for a profile wrapper."""
+import importlib
```

Comment:
Imports were put inside function intentionally. Profiler runs in the same process with specified program and any import adds memory overhead that should be taken account. These imports are used when profiling packages only.